sentinel-adapter/sentinel-spring-webmvc-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/spring/webmvc/AbstractSentinelInterceptor.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 1999-2019 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.csp.sentinel.adapter.spring.webmvc;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.web.servlet.HandlerInterceptor;
+import org.springframework.web.servlet.ModelAndView;
+
+/**
+ * @Author kaizi2009
+ * @description
+ * @date 2019-10-17
+ */
+public abstract class AbstractSentinelInterceptor implements HandlerInterceptor {
+
+    public static final String SPRING_MVC_CONTEXT_NAME = "spring_mvc_context";
+
+    @Override
+    public void postHandle(HttpServletRequest request, HttpServletResponse response, Object handler,
+                           ModelAndView modelAndView) throws Exception {
+    }
+
+    protected void setEntryContainerInReqeust(HttpServletRequest request, String name, EntryContainer entryContainer) {
+        Object attrVal = request.getAttribute(name);
+        if (attrVal != null) {
+            throw new SentinelSpringMvcException("Already exist attribute name '" + name + "' in request");
```

Comment:
It seems to me that throwing an exception directly is not very friendly.

Reply:
I will modify it.

Reply:
Done